ReactiveCocoa/Swift/Signal.swift

```diff
@@ -311,11 +311,11 @@ extension SignalType where Value: SignalProducerType, Error == Value.Error {
 	/// emitted from `signal` complete.
 	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
 	private func concat() -> Signal<Value.Value, Error> {
-		return Signal<Value.Value, Error> { [signal = self.signal] observer in
+		return Signal<Value.Value, Error> { observer in
```

Comment:
@iv-mexx `self` is the same object as `self.signal` at runtime. So capturing either `self.signal` or `self` will have the same effect.